CONTRIBUTE.md

```diff
@@ -5,7 +5,7 @@ before reporting things, and help us with the open ones.
 
 Guidelines:
 
-- read the [libp2p spec](https://github.com/ipfs/specs/tree/master/protocol/network)
+- read the [libp2p spec](https://github.com/ipfs/specs/tree/master/libp2p)
```

Comment:
*much table flipping*

All right, I'll fix this one, but there are *many* other places these'll need to be updated.